public/app/features/alerting/unified/components/contact-points/useContactPoints.ts

```diff
@@ -88,7 +88,7 @@ type K8sReceiver = ComGithubGrafanaGrafanaPkgApisAlertingNotificationsV0Alpha1Re
 
 const parseK8sReceiver = (item: K8sReceiver): GrafanaManagedContactPoint => {
   return {
-    id: item.metadata.uid || item.spec.title,
+    id: item.metadata.name || item.metadata.uid || item.spec.title,
```

Comment:
I would have expected `uid` to be the thing that the individual resource APIs are keyed off, but if it needs to be `name` then we can do that :+1: Though if there is no `name` set, we'll fall back to `uid`, and then trying to link to a contact point would fail. 
In practice, will `name` always be set? The types say they won't be

Reply:
Yes, the `name` is the only identifier that is supposed to be used by the client to manage resource. It uniquely identifies a resource of a specific kind. In contrast to name, `UID` identifies a resource across all kinds and namespaces that have ever been created. It has no practical use for the client other than detect whether a resource was deleted and re-created (such resources will have different UID).